### ISSUE TITLE
PWC-Net compared with Shi-Tomasi

### ISSUE THREAD

**Issue by gitzyong812** [deqings/PWC-Net#2]
Dear Dr. Sun,

I’ve read your paper “PWC-Net: CNNs for Optical Flow Using Pyramid, Warping, and Cost Volume”. 
I am wandering how the pre-trained PWC-Net is compared to traditional method like Shi-Tomasi algorithm in terms of accuracy?
I’ve read checked several papers of estimating optical with deep networks. None of them compare the results with traditional methods.
I don’t know why there is no such comparison.

By the way, for a new dataset without optical flow ground truth, how should we fine-tune the pre-trained PWC-Net?
Do you have any suggestions about this?
Thank you so much! 

Have a nice day!
